pypom/view.py

```diff
@@ -2,16 +2,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from selenium.common.exceptions import NoSuchElementException
-from selenium.webdriver.support.ui import WebDriverWait
+from .interfaces import IDriver
 
 
 class WebView(object):
 
-    def __init__(self, selenium, timeout):
-        self.selenium = selenium
+    def __init__(self, driver, timeout):
+        self.driver = driver
+        self.driver_adapter = IDriver(driver)
```

Comment:
Could you retain the `selenium` attribute so that this won't break existing users? Even better, raise a deprecation warning if it's accessed.